src/components/MemberArea/EditProfile.js

```diff
@@ -4,6 +4,17 @@ import model from './model';
 import Select from 'react-select';
 import './EditProfile.css';
 import { isMentor, fromMtoVM, fromVMtoM } from '../../helpers/user';
+import { Icons } from './socialMediaIcons';
+
+const IconMapper = {
```

Comment:
There is already mapping from channel to icon: https://github.com/Coding-Coach/find-a-mentor/blob/5170bc4e4b590678e65a3b246e02a910b1487ac1/src/channelProvider.js#L3